.github/workflows/visualize-repo.yml

```diff
@@ -0,0 +1,33 @@
+name: Render Visualization of Azure Service Operator v2
```

Comment:
This looks like it updates the visualization for v1 and v2? Maybe the title needs to be fixed?

Reply:
Fixed.